src/docs/product/crons/job-monitoring.mdx

```diff
@@ -23,6 +23,38 @@ The "Issues" section shows the issues created from this Cron Monitor. Issues are
 
 The table of recent check-ins lists previously scheduled jobs and their status.
 
+## Multiple environments
+
+<Alert level="info">
+    Environment support is still in it's early stages, the user experience may
+    have some kinks we've yet to work out. Feel free to email
+    [crons-feedback@sentry.io](mailto:crons-feedback@sentry.io) with any
+    feedback!
+</Alert>
```

Comment:
Do you think we really need this? Since in our main Crons page and set up page we already have a warning that Crons by itself is in beta.

```suggestion
```

Reply:
Yeah we can drop it I think you're right